higlass/tilesets.py

```diff
@@ -95,6 +95,17 @@ def cooler(filepath, uuid=None, **kwargs):
     )
 
 
+def bam(filepath, index_filename, uuid=None, **kwargs):
```

Comment:
The index filename is usually filepath + '.bai'. You could use it as a default.

Reply:
Done 👍